_community_members/xtansia.md

```diff
@@ -15,11 +15,12 @@ breadcrumbs:
       url: '/community/members/thomas-farr.html'
 github: xtansia
 twitter: xtansia
+linkedin: xtansia
 job_title_and_company: 'Software Development Engineer at AWS'
 personas:
   - author
 permalink: '/community/members/thomas-farr.html'
 redirect_from: '/authors/xtansia/'
 ---
 
```

Comment:
In the first file, we use "API Code Generator project", but here we use "API Code Generation project". Also, Bill's bio refers to "the Client API Generation project". Does this refer to the same or a different project?

Reply:
They all refer the same project:  API Code Generation for OS Clients (Generating the API implementation code in each OpenSearch client, which accounts for 99% of the client code). Sometimes we use Client CodeGen for short. I do see that this can lead to confusions.

Reply:
I changed the name of the project to be uniform.